packages/gatsby/src/schema/schema.js

```diff
@@ -840,6 +846,24 @@ const addCustomResolveFunctions = async ({ schemaComposer, parentSpan }) => {
   })
 }
 
+function attachTracingResolver({ schemaComposer }) {
+  schemaComposer.forEach(typeComposer => {
+    if (
+      typeComposer instanceof ObjectTypeComposer ||
+      typeComposer instanceof InterfaceTypeComposer
+    ) {
+      typeComposer.getFieldNames().forEach(fieldName => {
+        const field = typeComposer.getField(fieldName)
+        typeComposer.extendField(fieldName, {
+          resolve: field.resolve
+            ? tracingResolver(field.resolve)
+            : defaultTracingResolver,
```

Comment:
We have some logic in node-model which tests for presence of the resolver:

https://github.com/gatsbyjs/gatsby/blob/73a4b093e7b9dea53bd19f62f5758c0a87d14b14/packages/gatsby/src/schema/node-model.js#L591-L602

Can this change have some effects on it? I am not entirely sure.

Reply:
No, it should be same (or actually fixed) behaviour.